.eslintrc.json

```diff
@@ -0,0 +1,68 @@
+{
```

Comment:
Thanks for migrating to eslint. I'd like to note that on opening various ts files within vscode I'm seeing a lot of eslint errors popping up.
**suggestion**: Go through each individual ts file and apply the rules.

Reply:
Ok fixed, 1) I didn't see this I was in Webstorm. and 2) running `ng lint` did not pick up any lint warnings.. Perhaps something is still missing as I would of expected ng lint to display warnings you were seeing?

Reply:
I've included additional paths for linting in `angular.json`. It feels somewhat odd to have to define more paths while within the `"projects" -> "angular-reactive-validation"` scope. That could be related to the fact that when I introduced the project in a project, there was no true angular support for doing so yet.